warehouse/ingest-core/src/main/java/datawave/iterators/PropogatingIterator.java

```diff
@@ -319,11 +331,49 @@ public boolean validateOptions(Map<String,String> options) {
         
         PropogatingCombiner propAgg = null;
         
+        Map<String,Map<String,String>> allAggOptions = new HashedMap();
+        // make a first pass of the options to find any aggregator options and build them up for each aggregator
         for (Entry<String,String> familyOption : options.entrySet()) {
+            String option = familyOption.getKey();
+            // if this property specifies aggregator options skip
```

Comment:
It doesn't look like we're skipping if the property contains aggregator options.